packages/realm_dart/lib/src/cli/atlas_apps/baas_client.dart

```diff
@@ -29,14 +29,46 @@ class BaasAuthHelper {
   }
 }
 
+enum AppNames {
```

Comment:
Moved from `baas_helper` to make sure we have a single source of truth for the names.